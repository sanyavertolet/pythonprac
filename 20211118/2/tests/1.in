class Cls:
    num = Num()

print(Cls().num)
a, b = Cls(), Cls()
a.num = range(1, 100, 2)
print(a.num)
b.num = 'long as i can see the light'
print(a.num + b.num)
