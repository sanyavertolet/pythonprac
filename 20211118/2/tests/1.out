0
50
77
